code/game/machinery/spaceheater.dm

```diff
@@ -58,7 +58,7 @@
 
 /obj/machinery/space_heater/update_icon()
 	if(on)
-		icon_state = "sheater-[mode]"
+		icon_state = "sheater-[on ? "[mode]" : "off"]"
```

Comment:
bleh

Reply:
2 reasons: Because the heater would look like it's working when it was off for at least one cycle after its battery depletes. Because its what the electrolyzer does already, so I followed the lead of that JS file